__tests__/app.test.js

```diff
@@ -145,12 +145,47 @@ describe("GET /api/articles/articles_id/comments", () => {
         expect(allComments).toBeSortedBy("created_at", { descending: true });
       });
   });
-  test("GET 404: should give an appropriate error message when given an invalid endpoint", () => {
+  test("404: should give an appropriate error message when given an invalid endpoint", () => {
     return request(app)
       .get("/api/articles/1/banana")
       .expect(404)
       .then(({ body }) => {
-        expect(body.msg).toBe("Incorrect endpoint")
+        expect(body.msg).toBe("Incorrect endpoint");
+      });
+  });
+});
+
+describe.only("POST /api/articles/article_id/comments", () => {
+  test("201: should successfully post a comment and send the response back to the user", () => {
+    const toPostComment = {
+      username: "butter_bridge",
+      body: "comment",
+    };
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send(toPostComment)
+      .expect(201)
+      .then(({ body }) => {
+        const newComment = body.comment;
+        expect(newComment).toHaveProperty("comment_id");
+        expect(newComment).toHaveProperty("body");
+        expect(newComment).toHaveProperty("article_id");
+        expect(newComment).toHaveProperty("author");
+        expect(newComment).toHaveProperty("votes");
+        expect(newComment).toHaveProperty("created_at");
```

Comment:
Is there a more readable matcher we could use?

How could we check that the values are as we expect - we're POSTing this comment so we __know__ we want `body` to be "comment" and author to be "butter_bridge" for example as that's what we tried to POST. Is there any value that we couldn't be specific about?